plugins/content_distribution/providers/tvinci/TvinciDistributionPlugin.php

```diff
@@ -2,7 +2,8 @@
 /**
  * @package plugins.tvinciDistribution
  */
-class TvinciDistributionPlugin extends KalturaPlugin implements IKalturaPermissions, IKalturaEnumerator, IKalturaPending, IKalturaObjectLoader, IKalturaContentDistributionProvider, IKalturaConfigurator
+class TvinciDistributionPlugin extends KalturaParentContributedPlugin implements IKalturaPermissions, IKalturaEnumerator, IKalturaPending, IKalturaObjectLoader,
```

Comment:
Since the influence here is about how the MRSS is built it should be structured as an hierarchy implementation - since you want to influence "yourself"